From c4b9ce60a61d47a4f5244d7b86e8e30157149612 Mon Sep 17 00:00:00 2001
From: Jon Ringle <jon@ringle.org>
Date: Thu, 29 Oct 2009 16:27:16 +0100
Subject: [PATCH 21/21] Fix no previous prototype __ctype_* warnings

<...>/usr/lib/klibc/include/ctype.h:63: warning: no previous prototype for '__ctype_isascii'
<...>/usr/lib/klibc/include/ctype.h:68: warning: no previous prototype for '__ctype_isblank'
<...>/usr/lib/klibc/include/ctype.h:73: warning: no previous prototype for '__ctype_iscntrl'
<...>/usr/lib/klibc/include/ctype.h:78: warning: no previous prototype for '__ctype_isdigit'
<...>/usr/lib/klibc/include/ctype.h:83: warning: no previous prototype for '__ctype_isgraph'
<...>/usr/lib/klibc/include/ctype.h:89: warning: no previous prototype for '__ctype_islower'
<...>/usr/lib/klibc/include/ctype.h:94: warning: no previous prototype for '__ctype_isprint'
<...>/usr/lib/klibc/include/ctype.h:99: warning: no previous prototype for '__ctype_ispunct'
<...>/usr/lib/klibc/include/ctype.h:104: warning: no previous prototype for '__ctype_isspace'
<...>/usr/lib/klibc/include/ctype.h:109: warning: no previous prototype for '__ctype_isupper'
<...>/usr/lib/klibc/include/ctype.h:114: warning: no previous prototype for '__ctype_isxdigit'
<...>/usr/lib/klibc/include/ctype.h:123: warning: no previous prototype for '__ctype_toupper'
<...>/usr/lib/klibc/include/ctype.h:128: warning: no previous prototype for '__ctype_tolower'

Signed-off-by: Jon Ringle <jon@ringle.org>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 usr/include/ctype.h |   16 ++++++++++++++++
 1 files changed, 16 insertions(+), 0 deletions(-)

diff --git a/usr/include/ctype.h b/usr/include/ctype.h
index dfe1c46..ef7cac4 100644
--- a/usr/include/ctype.h
+++ b/usr/include/ctype.h
@@ -45,6 +45,22 @@ __extern int isxdigit(int);
 __extern int toupper(int);
 __extern int tolower(int);
 
+__must_inline int __ctype_isalnum(int);
+__must_inline int __ctype_isalpha(int);
+__must_inline int __ctype_isascii(int);
+__must_inline int __ctype_isblank(int);
+__must_inline int __ctype_iscntrl(int);
+__must_inline int __ctype_isdigit(int);
+__must_inline int __ctype_isgraph(int);
+__must_inline int __ctype_islower(int);
+__must_inline int __ctype_isprint(int);
+__must_inline int __ctype_ispunct(int);
+__must_inline int __ctype_isspace(int);
+__must_inline int __ctype_isupper(int);
+__must_inline int __ctype_isxdigit(int);
+__must_inline int __ctype_toupper(int);
+__must_inline int __ctype_tolower(int);
+
 extern const unsigned char __ctypes[];
 
 __must_inline int __ctype_isalnum(int __c)
-- 
1.6.5

